sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -124,8 +128,8 @@ object CommandUtils extends Logging {
           0L
       }
     }.getOrElse(0L)
-    val durationInMs = (System.nanoTime() - startTime) / (1000 * 1000)
-    logInfo(s"It took $durationInMs ms to calculate the total file size under path $locationUri.")
+    val durationInMs = (System.nanoTime() - startTime) / 1e6
+    logDebug(s"It took $durationInMs ms to calculate the total file size under path $locationUri.")
```

Comment:
$durationInMs is intended to round up to integer or long.
divided by 1e6 makes it double, may keep (1000 * 1000) please

Reply:
Thanks for pointing it out!

Reply:
Do you mean to change to debug level here?

Reply:
@srowen yes, it could be called in the "else" branch above, and one partition per log would be too much if the number of partitions is very large.
```
partitions.map { p =>
  calculateLocationSize(sessionState, catalogTable.identifier, p.storage.locationUri)
}.sum
```